GVRf/Sample/gvr-test-screenshot3D/src/org/gearvrf/testscreenshot3D/TestScreenshot3DScript.java

```diff
@@ -0,0 +1,108 @@
+/* Copyright 2015 Samsung Electronics Co., LTD
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gearvrf.testscreenshot3D;
+
+import org.gearvrf.GVRAndroidResource;
+import org.gearvrf.GVRContext;
+import org.gearvrf.testscreenshot3D.R;
+import org.gearvrf.GVRScene;
+import org.gearvrf.GVRSceneObject;
+import org.gearvrf.GVRScript;
+import org.gearvrf.GVRTexture;
+
+import android.util.Log;
+
+public class TestScreenshot3DScript extends GVRScript {
+
+    private static final float CUBE_WIDTH = 20.0f;
+    private GVRContext mGVRContext = null;
+
+    @Override
+    public void onInit(GVRContext gvrContext) {
+        mGVRContext = gvrContext;
+
+        GVRTexture front = mGVRContext.loadTexture(new GVRAndroidResource(
```

Comment:
I don't think that we should add any new samples that do synchronous loading - it would probably be better to use `loadFutureTexture()`